### PR TITLE
899 TC- WalkerColina -03

### PR DESCRIPTION
**Pull request by wcolina**

### CODE REVIEW

**Comment by wcolina** [comment]
**Test Case:**

image

**Automated Test Case Result:**

image

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class PPEList extends BasePage {
+
+    @FindBy(id = "btnNewPpe")
+    private WebElement newPPEButton;
+
+    @FindBy(className = "container")
+    private WebElement ppeContainer;
+
+    public PPEList() {
+        CommonEvents.isPresent(ppeContainer);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java]: usar CommonEvents.isVisible(ppeContainer);

File: src/test/resources/features/ppe/PPEList.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of ppe list
+Feature: PPEList
+
+  Scenario: Equipo de proteccion personal - Todos los datos de equipos de protección personal son mostrados en una lista
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEList.feature]: Corregir a minuscula "click"

File: src/test/resources/features/ppe/PPEList.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    And Click 'PPE' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/PPEList.feature]: Corregir a minuscula "click"

**Review by airmiky — COMMENTED** [review]
Revisar las observaciones por favor.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java (hunk 1 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEList.java]: Done.

File: src/test/resources/features/ppe/PPEList.feature (hunk 2 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEList.feature]: Done.

File: src/test/resources/features/ppe/PPEList.feature (hunk 3 above)
Comment (wcolina) [reply src/test/resources/features/ppe/PPEList.feature]: Done.

**Comment by wcolina** [comment]
Fix all comments and merged.

**Pull request merged**